test/functional/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -719,14 +719,11 @@ public void testEnvironmentVariableOrdering() {
 				argumentPositions.containsKey(openJ9JavaOptionsArg));
 		assertTrue(IBM_JAVA_OPTIONS+ SHOULD_COME_AFTER+OPENJ9_JAVA_OPTIONS, 
 				argumentPositions.get(ibmJavaOptionsArg).intValue() > argumentPositions.get(openJ9JavaOptionsArg).intValue());
-		if (!isJava8 || !isIBM) {
-			assertTrue(OPENJ9_JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_OPTIONS, 
-					argumentPositions.get(openJ9JavaOptionsArg).intValue() > argumentPositions.get(javaOptionsArg).intValue());
-			assertTrue(JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_TOOL_OPTIONS, 
-					argumentPositions.get(javaOptionsArg).intValue() > argumentPositions.get(javaToolOptionsArg).intValue());
-		} else {
-			assertTrue(OPENJ9_JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_TOOL_OPTIONS, 
-					argumentPositions.get(openJ9JavaOptionsArg).intValue() > argumentPositions.get(javaToolOptionsArg).intValue());
+		assertTrue(OPENJ9_JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_TOOL_OPTIONS, 
+				argumentPositions.get(openJ9JavaOptionsArg).intValue() > argumentPositions.get(javaToolOptionsArg).intValue());
+		if (!(isJava8 && isIBM)) {
+			assertTrue(JAVA_OPTIONS + SHOULD_COME_AFTER + IBM_JAVA_OPTIONS, 
+					argumentPositions.get(javaOptionsArg).intValue() > argumentPositions.get(ibmJavaOptionsArg).intValue());
```

Comment:
This doesn't seem right. Why would we expect a `!isIBM` VM to pay attention to `IBM_JAVA_OPTIONS`?

Reply:
IBM_JAVA_OPTIONS is and has always been parsed on OpenJ9. We added OPENJ9_JAVA_OPTIONS as the replacement, deprecating IBM_JAVA_OPTIONS, but it still works.

Reply:
The only difference between IBM Java 8 and other builds related to these changes is that IBM Java 8 doesn't parse _JAVA_OPTIONS and OpenJ9 builds do.

Reply:
I was looking at [`redirector.c`](https://github.com/eclipse-openj9/openj9/blob/master/runtime/redirector/redirector.c#L565) where it ignores `IBM_JAVA_OPTIONS` if `OPENJ9_JAVA_OPTIONS` is defined. Do you think that's inconsisten?

Reply:
It is weird that we check options in the redirector that way, and then process both environment variables in the VM. I'm not too concerned about it unless somebody complains. If we manage to switch IBM to use mixed builds then the redirector code becomes obsolete.

Reply:
The redirector is looking for options like `-Xnocompressedrefs`.